common/djangoapps/student/tests/test_bulk_email_settings.py

```diff
@@ -84,7 +86,7 @@ def test_email_authorized(self):
 
 
 @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
-class TestStudentDashboardEmailViewXMLBacked(ModuleStoreTestCase):
+class TestStudentDashboardEmailViewXMLBacked(SharedModuleStoreTestCase):
```

Comment:
Is there any performance benefit from this change or is it just for consistency reasons?